### ISSUE TITLE
Instance: The "New" (new reply) status message is not announced to assistive technology users | WCAG 2.2 Violation - There are status messages that appear within the interface that are not announced to assistive technology users.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#45]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [4.1.3 Status Messages](https://www.w3.org/TR/WCAG22/#status-messages)

#### Further information:

1. 

### Screenshot

![Screenshot showing a `.chat-new-message-indicator` element with the text 'New'. This element is displayed visually on screen, but the fact that a new reply has appeared is not actually announced to assistive technology users via live region.](https://github.com/user-attachments/assets/ebd2a070-7b23-4062-b009-c08fc7235608)

### Code Reference

```
<span class="chat-new-message-indicator" style="display: inline-flex; visibility: visible; opacity: 1;">New</span>
```
